src/test/regress/sql/chbenchmark_all_queries.sql

```diff
@@ -0,0 +1,722 @@
+SET citus.next_shard_id TO 1650000;
+CREATE SCHEMA chbenchmark_all_queries;
+SET search_path TO chbenchmark_all_queries;
+
+SET citus.enable_repartition_joins TO on;
```

Comment:
Wonder whether we should turn this on by default, or wait for the adaptive executor implementation.

Reply:
Don't know the reason for it being turned off, I always though this was to warn users about the slow performance once repartitions come in play.

Now the user needs to enable repartitions which might make them think about their sharding strategy more.